server/jira.js

```diff
@@ -268,6 +270,39 @@ Jira.verifyTicketOnDev = function(ticket) {
   Jira.makePostRequest(url, data);
 };
 
+Jira.reOpenAndCommentTicket = function(ticket, failReason) {
+  if (ticket.statusName === Config.jira.toDoStatusName) {
```

Comment:
This makes sense, since why transition a To Do ticket back to To Do. However, if multiple people fail the ticket only the first person's comment will show up. That's not necessarily a problem, but if you want all the comments to show up in Jira it might make sense to move this if statement to just surround the transition call below so the comment one still gets called.